Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -179,409 +135,363 @@ public void Dispose()
+                        _logger.LogError("Unable to parse bind override: {Entry}", entry);
                     }
-                }
+                    else
+                    {
+                        var replacement = parts[1].Trim();
+                        var ipParts = parts[0].Split("/");
+                        if (string.Equals(parts[0], "all", StringComparison.OrdinalIgnoreCase))
+                        {
+                            _publishedServerUrls[new IPData(IPAddress.Broadcast, null)] = replacement;
+                        }
+                        else if (string.Equals(parts[0], "external", StringComparison.OrdinalIgnoreCase))
+                        {
+                            _publishedServerUrls[new IPData(IPAddress.Any, new IPNetwork(IPAddress.Any, 0))] = replacement;
+                            _publishedServerUrls[new IPData(IPAddress.IPv6Any, new IPNetwork(IPAddress.IPv6Any, 0))] = replacement;
+                        }
+                        else if (string.Equals(parts[0], "internal", StringComparison.OrdinalIgnoreCase))
+                        {
+                            foreach (var lan in _lanSubnets)
+                            {
+                                var lanPrefix = lan.Prefix;
+                                _publishedServerUrls[new IPData(lanPrefix, new IPNetwork(lanPrefix, lan.PrefixLength))] = replacement;
+                            }
+                        }
+                        else if (IPAddress.TryParse(ipParts[0], out IPAddress? result))
+                        {
+                            var data = new IPData(result, null);
+                            if (ipParts.Length > 1 && int.TryParse(ipParts[1], out var netmask))
+                            {
+                                data.Subnet = new IPNetwork(result, netmask);
+                            }
 
-                return true;
+                            _publishedServerUrls[data] = replacement;
+                        }
+                        else if (TryParseInterface(ipParts[0], out var ifaces))
+                        {
+                            foreach (var iface in ifaces)
+                            {
+                                _publishedServerUrls[iface] = replacement;
+                            }
+                        }
+                        else
+                        {
+                            _logger.LogError("Unable to parse bind override: {Entry}", entry);
+                        }
+                    }
+                }
             }
-
-            return false;
         }
 
-        /// <inheritdoc/>
-        public bool HasRemoteAccess(IPAddress remoteIp)
+        private void ConfigurationUpdated(object? sender, ConfigurationUpdateEventArgs evt)
         {
-            var config = _configurationManager.GetNetworkConfiguration();
-            if (config.EnableRemoteAccess)
+            if (evt.Key.Equals("network", StringComparison.Ordinal))
```

Comment:
I believe we have a constant for this somewhere?

Reply:
Haven't found it if we have one.

Reply:
Maybe we should have one?

Reply:
I found it in `NetworkConfigurationStore.StoreKey` and replaced it with that.